### PR TITLE
[jsk_2016_01_baxter_apc] modify func get_mask_img() into one node

### PR DESCRIPTION
**Pull request by knorth55**
rbo_segmentation_node.py have three functions, mask image generation, spatial image generation and segmentation.
our idea is that it is easier to improve and debug if we split the rbo_segmentation_node.py into three node.

### CODE REVIEW

File: jsk_2016_01_baxter_apc/node_scripts/tf_bbox_to_mask.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        :type transform: Transform
+        """
+        # check frame_id of a point and transform just in case
+        assert camera_model.tf_frame == transform.header.frame_id
+        assert target_bin.bbox.header.frame_id == transform.child_frame_id
+
+        transformed_list = [
+                do_transform_point(corner, transform)
+                for corner in target_bin.corners]
+        projected_points = project_points(transformed_list, camera_model)
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/node_scripts/tf_bbox_to_mask.py]: undefined function? `project_points`

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/node_scripts/tf_bbox_to_mask.py]: Please rebase from #1427

File: jsk_2016_01_baxter_apc/node_scripts/tf_bbox_to_mask.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        self.shelf = {}
+        for bin_info in bin_info_arr_msg.array:
+            self.shelf[bin_info.name] = BinData(bin_info=bin_info)
+        self.pub = self.advertise('~output', Image, queue_size=1)
+
+    def subscribe(self):
+        self.sub = rospy.Subscriber('~input', CameraInfo, self._callback)
+
+    def unsubscribe(self):
+        self.sub .unregister()
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/node_scripts/tf_bbox_to_mask.py]: fix it later

**Comment by wkentaro** [comment]
Please describe at top of the PR why this is necessary.

**Comment by yuyu2172** [comment]
@wkentaro 
I think it is ready to be merged

**Pull request merged**